php/libraries/CouchDB.class.inc

```diff
@@ -488,8 +495,12 @@ class CouchDB
      * @return The page content (Without headers) returned by the
      *         CouchDB web server.
      */
-    function _postRelativeURL($url, $data, $op = 'POST', $printValue = false)
-    {
+    function _postRelativeURL(
+        string $url,
+        string $data,
+        string $op = 'POST',
+        bool $printValue = false
+    ) {
```

Comment:
I the page content returned in the form of a string? It would be useful to have a return type here.

Reply:
That makes sense to me but I can't confirm since I don't have experience with this library. @ridz1208 Maybe you know?

Reply:
It's a thin wrapper around _postURL. postURL is declared as     `function _postURL(
        string $url,
        array $data,
        array $headers = array(),
        bool $printValue = false
    ): string`

.. so yes, it should be a string. (But the signature of _postURL is also wrong, because while looking it up I noticed at one place it returns a null.)